Modelplace_Cloud_API/main.py

```diff
@@ -0,0 +1,50 @@
+from loguru import logger
+import time
+import json
+import os
+
+from argparser import parse_args
+from api_requests import login, process, task
+from utils import save_prediction_results, save_visualization_results
+
+
+def get_access_token(email:str, password:str) -> str:
+    login_data = login(email, password)
+    access_token = login_data['access_token']
+    logger.info('Successful login')
+    return access_token
+
+def get_task_id(model_id: int, input_file: str, access_token: str) -> json:
+    process_data = process(model_id, input_file, access_token)
+    task_id = process_data['task_id']
+    logger.info('Prediction run')
+    return task_id
```

Comment:
We don't need this function. This is just what process() returns.

Reply:
Actually, don't need to have this function because there is one in `api_requests.py`